leap-ces-ccda-orchestration-embedded/src/main/java/gov/hhs/onc/leap/ces/common/clients/model/card/Card.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
Don't we have this model POJOs in the java-client? Is there a reason we don't want to use that dependency?

Reply:
Java version issue was being thrown at runtime when our package was reference.  CONNECT 5.4 is build to JDK 1.7.  By going down this path, which is redundant, a single lib build to 1.7, containing all necessary is created.   Is redundant, but works for demonstration purpose at this time.

Reply:
I see. I wonder if we can move the java client library back to 1.7. I don't think there is anything there that depends on newer java features.